src/picoprobe_config.h

```diff
@@ -48,9 +48,19 @@
 
 // PIO config
 #define PROBE_SM 0
+
+#ifndef USE_WS2812
 #define PROBE_PIN_OFFSET 2
-#define PROBE_PIN_SWCLK PROBE_PIN_OFFSET + 0 // 2
-#define PROBE_PIN_SWDIO PROBE_PIN_OFFSET + 1 // 3
+#else
+#define PROBE_PIN_OFFSET 22 // Use the QTPy's QT STEMMA
+#endif
+
+#define PROBE_PIN_SWCLK PROBE_PIN_OFFSET + 0
+#define PROBE_PIN_SWDIO PROBE_PIN_OFFSET + 1
+
+#define NEO_SM 1
+#define NEO_PIN_PWR 11
+#define NEO_PIN_DAT 12
```

Comment:
These should probably be in `CMakeLists.txt` rather than this header file, just like `PICOPROBE_LED` ?
See also https://github.com/raspberrypi/pico-sdk/blob/master/src/boards/include/boards/adafruit_qtpy_rp2040.h

Reply:
Moved into `ws2812.h`.